File: VB/E3916.Web/DemoExportService.svc.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Linq
Imports System.Runtime.Serialization
Imports System.ServiceModel
Imports System.ServiceModel.Activation
Imports DevExpress.Data.Utils.ServiceModel

Namespace E3916.Web
	<SilverlightFaultBehavior> _
	Public Class DemoExportService
		Inherits DevExpress.Xpf.Printing.Service.ExportService
	End Class
End Namespace

File: CS/E3916.Web/DemoExportService.svc.cs
using System;
using System.Linq;
using System.Runtime.Serialization;
using System.ServiceModel;
using System.ServiceModel.Activation;
using DevExpress.Data.Utils.ServiceModel;

namespace E3916.Web {
    [SilverlightFaultBehavior]
    public class DemoExportService : DevExpress.Xpf.Printing.Service.ExportService {
    }
}
